content/applications/websites/website/configuration/cookies_bar.rst

```diff
@@ -2,75 +2,52 @@
-the related Cookie Policy page <cookies-bar/policy>`.
+.. note::
+   The :ref:`Cookie Policy <cookies-bar/policy>` page (/cookie-policy) is automatically created when
+   you enable the cookies bar.
 
 .. _cookies-bar/customization:
 
 Customization
 =============
 
-To adapt the display of the cookies bar, click :guilabel:`Edit` and select the :guilabel:`Cookies
-Bar` building block in the :guilabel:`Invisible Elements` at the bottom of the panel. Customize it
-using the options in the :guilabel:`Customize` tab in the edit panel. Three :guilabel:`Layouts` are
-available:
-
-- :guilabel:`Discrete`: thin bar
-- :guilabel:`Classic`: banner
-- :guilabel:`Popup`: you can change the popup's :guilabel:`Position` to the :guilabel:`Top`,
-  :guilabel:`Middle`, or :guilabel:`Bottom` of the screen.
+From your website, click :guilabel:`Edit`, go to the :guilabel:`Invisible Elements` section at the
+bottom of the panel, and click :guilabel:`Cookies Bar`. You can select a :guilabel:`Layout`, adjust
+the :guilabel:`Size` of the cookie bar, and enable :guilabel:`Backdrop` to gray out the page in the
+background when the cookies bar is displayed on the screen.
 
-You can also:
-
-- modify the :guilabel:`Size` of the cookies bar;
-- enable :guilabel:`Backdrop` to gray out the page in the background when the cookies bar is
-  displayed on the screen;
-- further customize the appearance of the cookies bar using :guilabel:`Block` and/or
-  :guilabel:`Column` customization options, which are available after clicking anywhere in the
-  building block.
+Click anywhere in the building block to further customize the appearance of the cookies bar using
+:guilabel:`Block`, :guilabel:`Column` and/or :guilabel:`Inline Text` customization options.
 
 To edit the contents of the cookies bar (i.e., the consent message), click directly in the building
 block.
 
-.. image:: cookies_bar/customization.png
-   :alt: Odoo Website's edit panel to customize the cookies bar.
-
 .. _cookies-bar/policy:
 
 Cookie policy
 =============
 
 When you enable the cookies bar for your website, Odoo creates the **Cookie Policy** page
-(`/cookie-policy`) with the list of essential and optional cookies. To access it, click the
-:guilabel:`Cookie Policy` hyperlink in the cookies bar or open the page from :menuselection:`Website
---> Site --> Pages`.
+(`/cookie-policy`) containing a non-exhaustive list of cookies, with their purpose and examples. To
+access it, click the :guilabel:`Cookie Policy` hyperlink in the cookies bar or open the page from
+:menuselection:`Website --> Site --> Pages`.
 
-The contents of the page can be adapted based on your website's features and characteristics if
-needed.
+You can edit the content of the page as per your needs by clicking the :guilabel:`Edit` button.
 
 .. tip::
-   You could add a link providing access to this page, in your website's footer, for example.
+   Add a link to this page in your website's footer, for example.
```

Comment:
```suggestion
   You could add a link to this page in your website's footer, for example.
```
I this case, since we're really *suggesting* something and not *giving instructions*, I think it might sound a little "softer" to keep "You could"